src/Microsoft.Azure.WebJobs.Extensions.Kafka/Listeners/KafkaListener.cs

```diff
@@ -349,5 +367,18 @@ public void Dispose()
         {
             Dispose(true);
         }
+
+        public ScaleMonitorDescriptor Descriptor => scaleMonitorDescriptor;
+
+        public Task<KafkaTriggerMetrics> GetMetricsAsync() => topicScaler.GetMetricsAsync();
+
+        async Task<ScaleMetrics> IScaleMonitor.GetMetricsAsync()
+        {
+            return await topicScaler.GetMetricsAsync();
```

Comment:
Call the above GetMetricsAsync method here rather than duplicating the call to topicScaler